File: lua/ht/plugins/tree-sitter/nvim-treesitter-endwise.lua
return {
  {
    "RRethy/nvim-treesitter-endwise",
    allow_in_vscode = true,
    lazy = true,
    ft = { "lua", "ruby", "vimscript" },
    dependencies = {
      "nvim-treesitter/nvim-treesitter",
    },
    config = function()
      require("nvim-treesitter.configs").setup {
        endwise = { enable = true },
      }
    end,
  },
}

File: src/conf/plugins/treesitter/nvim-treesitter-endwise.ts
import { Plugin } from "@core/model";

export const plugin = new Plugin({
  shortUrl: "RRethy/nvim-treesitter-endwise",
  lazy: {
    ft: ["lua", "ruby", "vimscript"],
    dependencies: ["nvim-treesitter/nvim-treesitter"],
    config: () => {
      luaRequire("nvim-treesitter.configs").setup({
        endwise: { enable: true },
      });
    },
  },
    allowInVscode: true,
});
